### ISSUE TITLE
Connecting from cli to web (for use in testing)

### ISSUE THREAD

**Issue by frankcarey** [blinkreaction/drude#42]
I can't seem to connect from the cli container so I can run my behat tests.

Ping web works (sort of) I get the box, but the pings don't respond.

```
ping web
PING web (198.105.254.228): 48 data bytes
^C--- web ping statistics ---
2 packets transmitted, 0 packets received, 100% packet loss

```

When I use curl, it tries to redirect me to localhost, but then localhost:80 doesn't respond at all.

```
$
root@cli:/var/www# curl http://web -i
HTTP/1.1 301 Moved Permanently
Server: nginx
Date: Tue, 21 Jul 2015 18:12:46 GMT
Content-Type: text/html
Transfer-Encoding: chunked
Connection: close
Location: http://localhost
Expires: Tue, 21 Jul 2015 18:12:45 GMT
Cache-Control: no-cache

<html>
<head><title>301 Moved Permanently</title></head>
<body bgcolor="white">
<center><h1>301 Moved Permanently</h1></center>
<hr><center>nginx</center>
</body>
</html>

root@cli:/var/www# curl http://localhost -i
curl: (7) couldn't connect to host
```

**Comment by achekulaev** [comment]
It's been a while. Now there is dsh behat command in develop branch

**Comment by lmakarov** [comment]
Should be fixed in [v1.0.0-rc1](https://github.com/blinkreaction/drude/releases/tag/v1.0.0-rc1)
